algorithms/utils/action_distributions.py

```diff
@@ -98,6 +109,9 @@ def _kl_inverse(self, other_log_probs):
     def _kl_symmetric(self, other_log_probs):
         return 0.5 * (self._kl(other_log_probs) + self._kl_inverse(other_log_probs))
```

Comment:
Can you just rewrite this for the case of a uniform prior?

Something like:
`uniform_prob = 1 / n`
`log_uniform_prob = log(1/n)`
`0.5 * (probs * (log_probs - log_uniform_prob) + uniform_prob * (log_uniform_prob - log_probs))`

Check if this actually works, should be better than the version based on tensor allocation.

Maybe write a test that confirms that this has the same output as the existing version.

Reply:
Confirmed the values to be the same. It only was missing `.sum(dim=-1)` which I added.